### DISCUSSION TITLE
Problem with jsonata in connection with homeassistant (e.g. notify)

### DISCUSSION THREAD

**Discussion by sirwastl** [zachowj/node-red-contrib-home-assistant-websocket#1645] [Q&A] [answered]
For months I've had an action node configured to send notifications to my IOS device. It's configured like this (JSONata):
It worked very well, so i recieved a message with the title an the current time

{
   "message":"{{ now().strftime('%d.%m.%Y %H:%M') }}",
   "title":"Machine is finished!"
}

Now I guess after an update, the code isn't working so i get this message:
![IMG_7577](https://github.com/user-attachments/assets/f74d5b4c-580b-402d-9f38-da0872c3386b)

Thanks for your help!

**Comment by zachowj** [comment, marked as answer]
That's actually a Jinja2 template that is passed to Home Assistant for rendering. There have been several reports that the notification service is no longer rendering templates correctly.

```
{
"message": $moment().format("d.m.Y H:M"),
"title":"Machine is finished!"
}
```

<https://momentjscom.readthedocs.io/en/latest/moment/04-displaying/01-format/>

**Reply by sirwastl** [reply]
Thanks. Is this just a bug or a new ‚Feature‘?

**Comment by sirwastl** [comment]
fixed
